doc/api/https.md

```diff
@@ -24,15 +24,23 @@ An [`Agent`][] object for HTTPS similar to [`http.Agent`][]. See
 [`https.request()`][] for more information.
 
 ### new Agent([options])
-
+<!-- YAML
+changes:
+  - version: REPLACEME
+  pr-url: FIXME
+  description: do not automatically set servername if the target host was
```

Comment:
I think these - pr-url and description - need to line up with version (i.e., add two more spaces.)